xcodeproj/internal/xcodeproj.template.bazelrc

```diff
@@ -99,6 +99,13 @@ build:rules_xcodeproj_swiftuipreviews --features=-swift.index_while_building
 build:rules_xcodeproj_tsan --config=rules_xcodeproj
 build:rules_xcodeproj_tsan --features=tsan
 
+### `--config=rules_xcodeproj_asan`
```

Comment:
Can you put this above the thread sanitizer?

Reply:
👍🏼